src/features/timeline/TimelineEditPage.js

```diff
@@ -30,10 +30,12 @@ const useStyles = makeStyles((theme) => ({
   },
 }))
 
-const CycleEditPage = ({ entryId }) => {
+const CycleEditPage = ({ entryId, path }) => {
   const classes = useStyles()
   const date = makeDate(entryId)
-
+  const redirectTo = path.includes("/calendar")
```

Comment:
Here I think it could be good to do ".." as path for `navigate`. It will move back one.

Reply:
I changed this, let me know if you still want this change